checkov/terraform/plan_runner.py

```diff
@@ -34,6 +34,11 @@ def run(self, root_folder=None, external_checks_dir=None, files=None, runner_fil
         if external_checks_dir:
             for directory in external_checks_dir:
                 resource_registry.load_external_checks(directory, runner_filter)
+        if root_folder:
+            root_folder = os.path.abspath(root_folder)
+            # Assume plan files will have this name by default
+            files = [f'{root_folder}/planfile.json']
```

Comment:
i think you can scann all json files and if it has a `  "terraform_version": "0.12.23", ` element - you can determine it's a plan file